src/components/ContactForm/index.tsx

```diff
@@ -0,0 +1,172 @@
+'use client';
+
+import React, { useRef, useState } from 'react';
+import emailjs from '@emailjs/browser';
+import { useTranslations } from 'next-intl';
+import { IoIosArrowDown } from 'react-icons/io';
+import { MdEmail } from "react-icons/md";
+import { FaPhoneAlt } from "react-icons/fa";
+import { IoLocation } from "react-icons/io5";
+
+export const ContactUs: React.FC = () => {
+  const form = useRef<HTMLFormElement>(null);
+  const t = useTranslations('button');
+  const b = useTranslations("contact_us")
+  const [isChecked, setIsChecked] = useState<boolean>(false);
+  const [selectedOption, setSelectedOption] = useState<string>('');
+  const [dropdownOpen, setDropdownOpen] = useState<boolean>(false);
+  const [showPopup, setShowPopup] = useState<boolean>(false);
+
+  const sendEmail = (e: React.FormEvent<HTMLFormElement>) => {
+    e.preventDefault();
+
+    if (!isChecked) {
+      alert('Please check the box to proceed');
```

Comment:
Kanske ändra till "Please confirm terms & conditions to proceed" för att understryka att det är vad de gör när de checkar boxen